srv/components/misc_pkgs/mocks/cortx/files/scripts/mock.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+import logging
+
+FORMAT = '%(asctime)s-%(message)s'
+LOGGING_FILE = '/tmp/mock.log'
+
+
+def main():
```

Comment:
In https://github.com/Seagate/cortx-prvsnr/blob/f256801af97a79255f33031b1cb7400bb741c8a6/srv/_modules/setup_conf.py#L82 we check if the command has a `--help` opion.
Do simulate the behavior to avoid failure.

Reply:
I think it is not needed here. When we execute the `mock.py` with the `--help` option, the script will log `--help` argument to the log file and will exit with a non-error exit code.